client/src/assets/icons/email.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!-- Generator: Gravit.io --><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="isolation:isolate" viewBox="493 86.492 241 151" width="241pt" height="151pt"><g><g><rect x="493" y="103.492" width="241" height="134" transform="matrix(1,0,0,1,0,0)" fill="rgb(73,189,158)"/><path d=" M 610.967 162.585 L 553.25 199.235 L 495.533 235.884 C 494.135 236.771 494.344 237.492 496 237.492 L 613.5 237.492 L 731 237.492 C 732.656 237.492 732.865 236.771 731.467 235.884 L 673.75 199.235 L 616.033 162.585 C 614.635 161.698 612.365 161.698 610.967 162.585 Z " fill="rgb(73,179,150)"/><path d=" M 610.967 178.399 L 553.25 141.749 L 495.533 105.1 C 494.135 104.213 494.344 103.492 496 103.492 L 613.5 103.492 L 731 103.492 C 732.656 103.492 732.865 104.213 731.467 105.1 L 673.75 141.749 L 616.033 178.399 C 614.635 179.286 612.365 179.286 610.967 178.399 Z " fill="rgb(77,206,172)"/></g></g></svg>
```

Comment:
I'm not sure about the extra empty space at the top. Shouldn't that make it a little difficult if we try to center it vertically? I think keeping a vertical symmetry would be better, what do you think? 
![Screenshot_20190702_093406_mod](https://user-images.githubusercontent.com/4089035/60522062-f1b5a780-9cad-11e9-8608-18d1a8a91c59.png)